modules/microshift-low-latency-kernelrt-conc.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-low-latency-kernelrt-conc_{context}"]
+= Using Red Hat Enterprise Linux for Real-Time
```

Comment:
```suggestion
= Using Red Hat Enterprise Linux for Real Time
```

Reply:
The "-"is not present here, just in real-time kernel (https://docs.redhat.com/en/documentation/red_hat_enterprise_linux_for_real_time/8/html-single/installing_rhel_8_for_real_time/index)